org.onionshare.OnionShare.yaml

```diff
@@ -0,0 +1,336 @@
+---
+app-id: org.onionshare.OnionShare
+command: onionshare
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+separate-locales: false
+finish-args:
+  - "--device=dri"
+  - "--share=ipc"
+  - "--share=network"
+  - "--socket=wayland"
+  - "--socket=x11"
+  - "--talk-name=org.freedesktop.Flatpak"
+  - "--talk-name=org.freedesktop.Notifications"
+  - "--talk-name=org.freedesktop.secrets"
+  - "--filesystem=home:ro"
+  - "--filesystem=~/OnionShare:create"
+  - "--filesystem=xdg-config/onionshare:create"
+cleanup:
+  - "/go"
+  - "/bin/scripts"
+modules:
+  - name: pyside2
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTS=OFF
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        sha256: f175c1d8813257904cf0efeb58e44f68d53b9916f73adaf9ce19514c0271c3fa
+        url: https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-5.15.1-src/pyside-setup-opensource-src-5.15.1.tar.xz
+      - type: shell
+        commands:
+          - mkdir -p /app/include/qt5tmp && cp -R /usr/include/Qt* /app/include/qt5tmp # https://bugreports.qt.io/browse/PYSIDE-787
+          - sed -i 's|\(--include-paths=\)|\1/app/include/qt5tmp:|' sources/pyside2/cmake/Macros/PySideModules.cmake
+  - name: tor
+    buildsystem: simple
+    build-commands:
+      - "./configure --prefix=${FLATPAK_DEST}"
+      - make
+      - make install
+    sources:
+      - type: archive
+        sha256: 22cba3794fedd5fa87afc1e512c6ce2c21bc20b4e1c6f8079d832dc1e545e733
+        url: https://dist.torproject.org/tor-0.4.5.6.tar.gz
+    modules:
+      - name: libevent
+        buildsystem: simple
+        build-commands:
+          - "./configure --prefix=${FLATPAK_DEST}"
+          - make
+          - make install
+        sources:
+          - type: archive
+            url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+            sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+  - name: obfs4proxy
+    buildsystem: simple
+    build-options:
+      env:
+        GOBIN: "/app/bin/"
+        GO111MODULE: "off"
+    build-commands:
+      - ". /usr/lib/sdk/golang/enable.sh; GOPATH=$PWD go install gitlab.com/yawning/obfs4.git/obfs4proxy"
```

Comment:
Does it actually work? Flathub runs builds offline, https://github.com/flatpak/flatpak-builder-tools/tree/master/go-get can be used to generate manifest automatically.